app/views/main/index.html

```diff
@@ -1,2 +1,6 @@
 <h1>Main page</h1>
 
+<ul>
+  <li> Armando Fox</li>
```

Comment:
Let's cleanup this space.

```suggestion
  <li>Armando Fox</li>
```